hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminServer.java

```diff
@@ -177,10 +177,7 @@ private void checkServersAndTables(Set<Address> servers, Set<TableName> tables,
               + " does not exist.");
     }
     RSGroupInfo srcGrp = new RSGroupInfo(tmpSrcGrp);
-    if (srcGrp.getName().equals(targetGroupName)) {
```

Comment:
on line 167, need to change the javadoc "@throws IOException if nulls or if servers and tables not belong to the same group"
this is not accurate anymore :)

Reply:
Hi, @xcangCRM , for` the javadoc "@throws IOException if nulls or if servers and tables not belong to the same group"`, I think it means the check of the source group of moved servers or tables... If there exists two or more source groups of the the servers and tables, it will throw exception.  The check is on line 184 to 200.

Reply:
ok thanks!